### ISSUE TITLE
Forward to URL Open in External Browser

### ISSUE THREAD

**Issue by NielStuartParker** [markusfisch/BinaryEye#505]
Hello,

I'm trying to figure out if it's possible to open an url in an external browser while also appending a string to the end of the barcode result. Something like:

www.example.com/pages/{BARCODE}-CUSTOMSTRING.aspx

Apologies if I'm just not understanding the exisiting documentation. I've been playing around with it this morning, but can't get it to work.

This is an excellent app and any help you might off is greatly appreciated!

Best regards,
Niel

**Comment by markusfisch** [comment]
Hi Neil,

Yes, this is possible. In the settings, look for the "Forwarding" section where you will find these three options:

![Screenshot_20241022-211102.png](https://github.com/user-attachments/assets/39cfbabb-01a8-4299-bdb8-8b356d439b32)

You would need to enable forwarding (the first option), enter your URL (the second) and choose "Open in external browser" (in the third).

Please note that the barcode content will be appended to the URL if it ends with an "=". So the barcode contents will always be at the end of the string.

And thanks for using Binary Eye!

Greetings 
Markus

**Comment by NielStuartParker** [comment]
Is it only possible to insert the barcode at the end of the string?

I was hoping that I could insert the barcode anywhere in the string.

**Comment by markusfisch** [comment]
Yes, the barcode can only be appended with the open in external browser option. But that shouldn't be a problem when you're in control of the server. It's always possible to process a query parameter no matter what technology you're using. So if you make the URL end with `?content=` you will be able to use `content` in your solution.
